app/javascript/packs/inherited-proofing-consent.js

```diff
@@ -0,0 +1,8 @@
+const form = document.querySelector('.js-consent-continue-form');
+
+if (form) {
+  const input = document.createElement('input');
+  input.type = 'hidden';
+  input.name = 'skip_upload';
+  form.appendChild(input);
+}
```

Comment:
Since we do not have an upload step, we may not even need this, as this appears to be associated with uploading documents?:

app/services/idv/flows/doc_auth_flow.rb
app/services/idv/steps/agreement_step.rb

`genemangelo@FCOH2J-G1AFLVDF ＞git grep skip_upload                                                                                                                      (3.0.4)
app/controllers/idv/doc_auth_controller.rb:      return if params[:step] != 'upload' || !flow_session || !flow_session[:skip_upload_step]
app/javascript/packs/document-capture-welcome.js:  input.name = 'skip_upload';
app/javascript/packs/inherited-proofing-consent.js:  input.name = 'skip_upload';
app/services/idv/actions/redo_document_capture_action.rb:        unless flow_session[:skip_upload_step]
app/services/idv/steps/agreement_step.rb:        skip_to_capture if params[:skip_upload]
app/services/idv/steps/agreement_step.rb:        flow_session[:skip_upload_step] = true
app/services/idv/steps/upload_step.rb:        !!flow_session[:skip_upload_step]`

Reply:
Looks like this was resolved.